src/main/java/org/pih/petl/job/config/JobConfigReader.java

```diff
@@ -143,7 +143,14 @@ public Integer getInt(Integer defaultValue, String...keys) {
     public Boolean getBoolean(Boolean defaultValue, String... keys) {
         JsonNode n = get(keys);
         if (n != null) {
-            return n.asBoolean();
+            if (n.isBoolean()) {
+                return n.asBoolean();
+            }
+            String stringVal = getString(n);
+            if (StringUtils.isNotBlank(stringVal)) {
+                return Boolean.parseBoolean(stringVal);
+            }
+            return false;
```

Comment:
Discovered this bug when trying to use a parameter value in place of a boolean configuration value.